tasks/IKF.cpp

```diff
@@ -98,6 +98,9 @@ void IKF::imu_samplesTransformerCallback(const base::Time &ts, const ::base::sam
             if (correction_idx == correction_numbers)
             {
                 acc = correctionAcc / (double)correction_numbers;
+
+                acc_body = acc; // save this to be output later after being corrected
```

Comment:
So currently the corrected acceleration measurement is written out with the same speed of the incoming samples,
+1 it should be exactly like this
but the value changes only if a new measurements is integrated.
 -1 this is not complete true. The filtered acc value changes when a new acc arrives into the input port, it is the correction (estimation in the acc bias in the state vector) which changes when a new measurement is integrated at the measurement rate.